src/gwt/acesupport/acemode/r_highlight_rules.js

```diff
@@ -1275,6 +1275,54 @@ define("mode/r_highlight_rules", ["require", "exports", "module"], function(requ
     }]);
 
     this.normalizeRules();
+
+    // allow for multi-line strings in YAML comments
+    this.$rules["yaml-multiline-string"].unshift({
+      regex: /^(#[|])(\s*)/,
+      onMatch: function (value, state, stack) {
+
+        // if the indent has decreased relative to what
+        // was used to start the multiline string, then
+        // exit multiline string state
+        var next = stack[0];
+        var indent = stack[1];
+        debugger;
```

Comment:
@kevinushey didn't notice this before I approved but is this a stray debugger call?

Reply:
Oops, you're right -- I'll clean that up!